test/extensions.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
```

Comment:
We don't need `"use strict"` here, it's the default in ES modules!

Reply:
Oh then I found the test I gave you still has it. Anyway it's not needed 😛

Reply:
Removed in commit [f12005e](https://github.com/w3c/webidl2.js/pull/731/commits/f12005ee66c058dd155c628d63168043b04de1b1)